redaxo/src/addons/mediapool/assets/mediapool.js

```diff
@@ -189,18 +188,18 @@ $(document).ready(function () {
             if (width != 0)
                 img.attr('width', width);
 
-            div.slideDown("fast");
+            div.stop(true, false).slideDown("fast");
         }
         else
         {
-            div.slideUp("fast");
+            div.stop(true, false).slideUp("fast");
         }
     }
 
     // Medialist preview neu anzeigen, beim wechsel der auswahl
     $('body')
         .on('click', '.rex-js-widget-medialist.rex-js-widget-preview', rexShowMediaPreview)
-        .on('mousemove', '.rex-js-widget-media.rex-js-widget-preview, .rex-js-widget-medialist.rex-js-widget-preview', rexShowMediaPreview)
+        .on('mouseenter', '.rex-js-widget-media.rex-js-widget-preview, .rex-js-widget-medialist.rex-js-widget-preview', rexShowMediaPreview)
         .on('mouseleave', '.rex-js-widget-media.rex-js-widget-preview, .rex-js-widget-medialist.rex-js-widget-preview', function() {
             var div = $('.rex-js-media-preview', this);
             if(div.css('height') != 'auto')
```

Comment:
Nein, beim Ausklappen reicht es in diesem Fall, weil es nur die beiden Animationen Ein- und Ausklappen gibt, so dass bei jedem Wechsel zwangsläufig vorher gestoppt wird.